scalableminds/webknossos
# Clean up temporary upload files

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- start upload, but do not complete it
- restart wk
- => temporary upload file should be gone

### Issues:
- fixes #4884 

------
